src/site/layouts/health_care_local_facility.drupal.liquid

```diff
@@ -192,15 +192,10 @@
             </section>
           {% endif %}
 
-          <h2 id="our-patient-satisfaction-scores"
-              class="vads-u-margin-top--4 vads-u-font-size--lg small-screen:vads-u-font-size--xl">
-            Veteran satisfaction with appointment wait times at this location
-          </h2>
```

Comment:
Optional: We could add a unit test to confirm that the template does not contain this header, since it is now controlled by the React widget.